test/controllers/project_skill_controller_test.exs

```diff
@@ -1,14 +1,16 @@
 defmodule CodeCorps.ProjectSkillControllerTest do
-  use CodeCorps.ApiCase
+  use CodeCorps.ApiCase, resource_name: :project_skill
 
   alias CodeCorps.ProjectSkill
   alias CodeCorps.Repo
 
-  defp build_payload, do: %{ "data" => %{"type" => "project-skill", "attributes" => %{}}}
+  @valid_attrs %{}
```

Comment:
I think the contrapositive here is again instructive: is `@valid_attrs` useful _at all_?

Seems like you'd be better off assembling the `attrs` as a map, without a `Map.merge` operation, inside the tests where you need those relationships.